lib/jxl/enc_fast_lossless.cc

```diff
@@ -948,7 +952,7 @@ FJXL_INLINE void StoreNeonAbove14(const uint32_t* nbits_tok,
 
 void EncodeHybridUint000(uint32_t value, uint32_t* token, uint32_t* nbits,
                          uint32_t* bits) {
-  uint32_t n = 31 - __builtin_clz(value);
+  uint32_t n = value ? 31 - __builtin_clz(value) : 32;
```

Comment:
I think there's a function that does exactly this in base/bits.h, might be nicer to use that one?

Reply:
There is such a function, but I'd like enc_fast_lossless to be usable in a drop-in way, so I'd prefer to avoid using it.

Reply:
Fair enough.